src/main/java/appeng/block/misc/BlockSecurity.java

```diff
@@ -66,4 +72,38 @@ public boolean onActivated(World w, int x, int y, int z, EntityPlayer p, int sid
 		return false;
 	}
 
+	public static class Listener
+	{
+		public static final Listener instance = new Listener();
+
+		@SubscribeEvent
+		public void listenBreakBlock(BlockEvent.BreakEvent event)
+		{
+			// If logging is disabled, the listener isn't registered
+
+			// if ( ! AEConfig.instance.isFeatureEnabled( AEFeature.LogSecurityTerminalBreak ) )
+			//	return;
+
+			// Only listen to events for breaking BlockSecurity
+			if ( ! ( event.block instanceof BlockSecurity ) )
```

Comment:
`instanceof` doesn't get NPEs, and the second use is a cast in order to assign it to the variable. So I think it's fine here.